chess/src/Components/ChooseRoom.js

```diff
@@ -19,7 +19,11 @@ class ChooseRoom extends React.Component {
             },
           }}
         >
-          <button id="bathroom" value="bathroom">
+          <button
+            id="bathroom"
+            value="bathroom"
+            disabled={!this.props.bathroom}
```

Comment:
Should we allow users to spectate other matches?

Reply:
no because it might reset the board if the third person exits the room before the game is finished